app/src/editor/canvas/components/ModuleSearch.pcss

```diff
@@ -83,6 +95,31 @@
   color: #CDCDCD;
 }
 
+.ModuleSearch .Input .ClearButton {
```

Comment:
Clear button could have a `pointer` cursor.

Reply:
Questionable, as discussed in this 4 part series: https://medium.com/simple-human/buttons-shouldnt-have-a-hand-cursor-b11e99ca374b

Reply:
Mmm, interesting read. I guess we're going with a button so let leave the default cursor. 👍